src/main/java/org/skriptlang/skript/lang/script/Script.java

```diff
@@ -103,46 +103,56 @@ public boolean suppressesWarning(ScriptWarning warning) {
 	private final Map<Class<? extends ScriptData>, ScriptData> scriptData = new ConcurrentHashMap<>(5);
 
 	/**
-	 * Clears the data stored for this script.
-	 */
-	public void clearData() {
-		scriptData.clear();
-	}
-
-	/**
+	 * <b>This API is experimental and subject to change.</b>
 	 * Adds new ScriptData to this Script's data map.
 	 * @param data The data to add.
 	 */
+	@ApiStatus.Experimental
```

Comment:
If you expect these methods to change it would probably have been better to put them in some kind of 'unsafe' class, rather than putting warnings on these.

Reply:
These are expected to change in https://github.com/SkriptLang/Skript/pull/5552 which is now targeting 2.8 (hence these annotations)

I'm not expecting a big impact (if any) on developers